src/actions/arcActions.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { getDAOs, createDAO, CreateDAOAction, createProposal, CreateProposalAction } from "./arcActions";
+import { Dispatch } from "react-redux";
+import { IRootState } from "reducers";
+import * as Arc from "@daostack/arc.js";
+import { ActionTypes } from "reducers/arcReducer"
+import { mockStore } from "../configureStore";
+import axios from "axios";
+import MockAdapter from 'axios-mock-adapter';
+import { Web3 } from "@daostack/arc.js";
+import { AsyncActionSequence } from "./async";
+import { push, LOCATION_CHANGE, CALL_HISTORY_METHOD } from "react-router-redux";
+import { IDaoState, IAccountState } from "reducers/arcReducer";
+import { member } from "components/ViewDao/ViewDao.scss";
+
+describe('arcActions', () => {
+
+  const store = mockStore();
```

Comment:
it didnt seem like this was working last time i worked on the tests branch, are you sure it is? it seemed like the tests were still using the regular store